src/Core/Services/OpenAPI/OpenApiDocumentor.cs

```diff
@@ -683,10 +683,10 @@ private string GetEntityRestPath(string entityName)
             string entityRestPath = entityName;
             EntityRestOptions entityRestSettings = _runtimeConfig.Entities[entityName].Rest;
 
-            if (!string.IsNullOrEmpty(entityRestSettings.Path) && entityRestSettings.Path.StartsWith('/'))
```

Comment:
by removing `entityRestSettings.Path.StartsWith('/')` from the conditional, i avoid improperly setting the PATH due to the edge case of when a user does not define their custom rest path with a starting slash `/`